modules/microshift-dynamic-provisioning-available-plugins.adoc

```diff
@@ -0,0 +1,77 @@
+// Module included in the following assemblies
+//
+// * storage/dynamic-provisioning.adoc
+// * post_installation_configuration/storage-configuration.adoc
+
+[id="available-plug-ins_{context}"]
+= Available dynamic provisioning plug-ins
+
+{product-title} provides the following provisioner plug-ins, which have
+generic implementations for dynamic provisioning that use the cluster's
+configured provider's API to create new storage resources:
+
+
+[options="header",cols="1,1,1"]
+|===
+
+|Storage type
+|Provisioner plug-in name
+|Notes
+
+|{rh-openstack-first} Cinder
+|`kubernetes.io/cinder`
+|
+
+|{rh-openstack} Manila Container Storage Interface (CSI)
+|`manila.csi.openstack.org`
+|Once installed, the OpenStack Manila CSI Driver Operator and ManilaDriver automatically create the required storage classes for all available Manila share types needed for dynamic provisioning.
+
+|AWS Elastic Block Store (EBS)
+|`kubernetes.io/aws-ebs`
+|For dynamic provisioning when using multiple clusters in different zones,
+tag each node with `Key=kubernetes.io/cluster/<cluster_name>,Value=<cluster_id>`
+where `<cluster_name>` and `<cluster_id>` are unique per cluster.
+
+|Azure Disk
+|`kubernetes.io/azure-disk`
+|
+
+|Azure File
+|`kubernetes.io/azure-file`
+|The `persistent-volume-binder` service account requires permissions to create
+and get secrets to store the Azure storage account and keys.
+
+|GCE Persistent Disk (gcePD)
+|`kubernetes.io/gce-pd`
+|In multi-zone configurations, it is advisable to run one {product-title}
+cluster per GCE project to avoid PVs from being created in zones where
+no node in the current cluster exists.
+
```

Comment:
None of these are supported on MicroShift.

Reply:
Removed the entire module